include/h2o/header.h

```diff
@@ -106,7 +106,7 @@ ssize_t h2o_set_header_token(h2o_mem_pool_t *pool, h2o_headers_t *headers, const
 /**
  * deletes a header from list
  */
-ssize_t h2o_delete_header(h2o_headers_t *headers, ssize_t cursor);
+void h2o_delete_header(h2o_headers_t *headers, ssize_t cursor);
```

Comment:
Yes, fixed.
Do we need consider keeping API stable from now on?
And for this function the return value seems useless because i can not image how the library user can use the return value(perhaps call h2o_delete_header again with return value?).
Anyway, leave it as-is.